### PR TITLE
.NET Core Desktop 3.0.1 + .NET Core 3.0.1 Portable

### PR DESCRIPTION
**Pull request by Heteroculturalism**

### CODE REVIEW

**Comment by riezebosch** [comment]
I updated portable to auto-update itself. Please rework your commit(s) to exclude that changes.

**Comment by Heteroculturalism** [comment]
Done.

**Comment by jberezanski** [comment]
Sorry about the long wait, @Heteroculturalism. I'd like to see this PR through to the end.

There are several other commits intertwined with yours here; we should get rid of them. Also, discard the changes to `dotnetcore-runtime.portable`, because they are long outdated with respect to `master`. But first, let's discuss design.

You created these three packages:
`dotnetcore-desktop` -> `dotnetcore-desktop-runtime` -> `dotnetcore-desktop-runtime.install`
(arrows=dependencies)

The splitting of runtime packages into `.install` and `.portable` was initially thought valuable because it appeared that only `.portable` would install side by side. It turned out not to be the case and I see no real reason to keep maintaining `.portable` - I'd like to fold the existing `dotnetcore-runtime.install` into simply `dotnetcore-runtime`. I believe we can skip `.install` for the desktop runtime.

I also see no need to distinguish between `dotnetcore-desktop` and `dotnetcore-desktop-runtime`. I mean, there is no other desktop component than the runtime. Less packages = lower maintenance burden and less chance of moderation failures.

So we arrived at a single package. Ideally, its id should match the convention of other packages (ignoring the `dotnetcore` metapackage), which is `dotnetcore-<component>` (a single dash). How about `dotnetcore-desktopruntime`?

File: dotnetcore-desktopruntime/tools/data.ps1 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿@{
+    PackageName = 'dotnetcore-desktop-runtime.install'
```
Comment (jberezanski) [inline dotnetcore-desktopruntime/tools/data.ps1:2]: The previous package name is still here. I'll fix this myself.
```suggestion
    PackageName = 'dotnetcore-desktopruntime'
```

**Comment by jberezanski** [comment]
Thank you for your contribution!

**Pull request merged**